.travis.yml

```diff
@@ -31,7 +31,7 @@ env:
   - INSTANCE=volume-ubuntu-1604
   - INSTANCE=resources-debian-9
   - INSTANCE=resources-centos-7
-  - INSTANCE=resources-fedora-31
```

Comment:
it's building now. I forgot about that one. There is fedora-latest though which avoids the need for bumping things

Reply:
great idea - just made this change